data_validation/config_manager.py

```diff
@@ -150,6 +150,17 @@ def append_query_groups(self, grouped_column_configs):
             self.query_groups + grouped_column_configs
         )
 
+    @property
+    def custom_query_type(self):
+        """Return SQL Query File from Config"""
+        return self._config.get(consts.CONFIG_CUSTOM_QUERY_TYPE, [])
```

Comment:
Shouldn't the default value here be 'None' instead of an empty list? The custom_query_type should only allow two string values: "row" or "column"

Reply:
Done